.azure/pipelines/Phonebook.preview.yml

```diff
@@ -0,0 +1,193 @@
+pr: 
+  branches:
+    include:
+      - master
+
+variables:
+  image_repo: danielhabenicht/phonebook
+  image_tag: pr-$(System.PullRequest.PullRequestNumber) 
+  helm_artifact_name: HelmPhonebookPackage
+
+  jobs:
+    - job: build_phonebook_frontend
+      displayName: 'Phonebook Frontend'
+      pool:
+        vmImage: 'ubuntu-16.04'
+      steps:
+      - task: UseNode@1
+        inputs:
+          version: '11.x'
+
+      - script: node version.js $(image_tag) $(Build.SourceVersion) $(Build.SourceVersion)
```

Comment:
why you must use the script tasks? Can we use the specific node or docker task? That special task can handle many edge cases.

Reply:
I will use node task tomorrow

Reply:
There is no node task in azure devops. I am not using the docker task because it brings in overhead by tagging additional information to the image.